### PR TITLE
Release 1.17.3

### PR DESCRIPTION
**Pull request by Lee-W**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+1.17.3 (2023-08-07)
+-------------------
+
+Bug Fixes
+"""""""""
+* Raise ``AirflowSkipExecption`` for sensos when ``soft_fail`` is paassed (#1276)
```
Comment (pankajkoti) [inline CHANGELOG.rst:9]:
```suggestion
* Raise ``AirflowSkipExecption`` for sensors when ``soft_fail`` is set (#1276)
```

**Review by pankajkoti — APPROVED** [review]

**Comment by Lee-W** [comment]
I think we'll need to wait for @phanikumv 's help on setting up the token before we merge this one

**Comment by pankajastro** [comment]
Do we need to mention Python3.11  that the hive provider is not supported?

**Review by pankajastro — APPROVED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+1.17.3 (2023-08-07)
+-------------------
+
+Bug Fixes
+"""""""""
+* Raise ``AirflowSkipExecption`` for sensors when ``soft_fail`` is set (#1276)
```
Comment (phanikumv) [inline CHANGELOG.rst:9]:
```suggestion
* Raise ``AirflowSkipExecption`` for sensors when ``soft_fail`` is set to True(#1276)
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+1.17.3 (2023-08-07)
+-------------------
+
+Bug Fixes
+"""""""""
+* Raise ``AirflowSkipExecption`` for sensors when ``soft_fail`` is set to True(#1276)
```
Comment (phanikumv) [inline CHANGELOG.rst:9]:
```suggestion
* Raise ``AirflowSkipException`` for sensors when ``soft_fail`` is set to True(#1276)
```

**Comment by Lee-W** [comment]
> Do we need to mention Python3.11 that the hive provider is not supported?

Sure. Not sure where it should be included. I added an `Extra Note` section for it.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+1.17.3 (2023-08-07)
+-------------------
+
+Bug Fixes
+"""""""""
+* Raise ``AirflowSkipException`` for sensors when ``soft_fail`` is set to True(#1276)
+* ``S3KeyTrigger``: Call S3 sensor check_fn only with file size attribute (#1278)
+* ``HttpSensorAsync``: Fix incorrect warning when poke_interval is passed (#1281)
+
+Extra Note
```
Comment (phanikumv) [inline CHANGELOG.rst:13]:
```suggestion
Note
```

**Pull request merged**